instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -9,51 +9,34 @@ module Instrumentation
     module AwsSdk
       # Generates Spans for all interactions with AwsSdk
       class Handler < Seahorse::Client::Handler
-        def call(context)
-          span_name = get_span_name(context)
-          attributes = get_span_attributes(context)
+        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+          return super unless context
```

Comment:
I noticed that `context&` was being invoked in multiple places.  I think without context we don't have a service name or an operation name, which means we don't have a span name.  That's likely indication of a bigger issue in the call chain here.  Regardless I think it's safer to return a call to super if we cannot collect any of the information to generate a span.  Happy to hear differing opinions here, such as potentially falling back to some ultra generic span naming without any attributes.